sql-gremlin/src/test/java/org/twilmes/sql/gremlin/adapter/GremlinSqlBaseTest.java

```diff
@@ -151,20 +151,12 @@ public void assertColumns(final List<String> actual, final List<String> expected
 
         SqlGremlinTestResult(final SqlGremlinQueryResult sqlGremlinQueryResult) throws SQLException {
             columns = sqlGremlinQueryResult.getColumns();
-            Object res;
             do {
-                try {
-                    res = sqlGremlinQueryResult.getResult();
-                } catch (final SQLException e) {
-                    if (e.getMessage().equals(SqlGremlinQueryResult.EMPTY_MESSAGE)) {
-                        break;
-                    } else {
-                        throw e;
-                    }
-                }
-                if (res instanceof List<?>) {
-                    this.rows.add((List<?>) res);
+                final List<?> res = sqlGremlinQueryResult.getResult();
+                if (res instanceof SqlGremlinQueryResult.EmptyResult) {
+                    break;
                 }
+                this.rows.add(res);
             } while (true);
```

Comment:
Nit: wouldn't this be better as a `while NOT res instanceof EmptyResult` instead of a do while true?

Reply:
@simonz-bq Lol, how did I miss this 🙃  will fix that. Thanks